src/tracker.rs

```diff
@@ -61,6 +62,7 @@ pub fn parse_torrent_file(file_path: &str) -> Result<Metafile> {
     file.read_to_end(&mut buffer)?;
 
     let meta_file = serde_bencoded::from_bytes::<Metafile>(&buffer[..])?;
+    trace!("Meta file has been read");
```

Comment:
We don't need this because we already logged the parsed meta file in the top level